Dockerfile.base

```diff
@@ -60,14 +60,19 @@ COPY container/elliott-settings.yaml /home/"$USERNAME"/.config/elliott/settings.
 WORKDIR /workspaces/art-dash
 
 USER "$USER_UID"
+
+#Setup uv package manager
+RUN curl -LsSf https://astral.sh/uv/install.sh | env UV_INSTALL_DIR="/home/$USERNAME/.local" sh \
+    && uv venv --python 3.11 ../.venv
```

Comment:
```suggestion

# Setup uv package manager
RUN pip3 install uv
```

Reply:
When installing `uv` with pip, `uv self update` is disabled and it has to be updated with `pip`. I wanted to avoid having to update `pip` to then using that to update `uv` but not really sure if my thinking there is correct. What do you think?

As for installing the virtual environment in `workspaces/venv` instead of `workspaces/art-dash/venv` - I did it this way because in the art-dashboard-server README, the `podman run ...` command specified also includes 
`-v "$OPENSHIFT/art-dashboard-server":/workspaces/art-dash:cached` which can potentially overwrite the `venv` we've setup while building the container (if the user has their own `art-dashboard-server/venv` in their local repo)